policies/ControllerAffinityNodeSelector/README.md

```diff
@@ -0,0 +1,47 @@
+# Affinity Node Selector
+
+This Policy allows setting a key and value for `nodeSelector` when assigning pods to nodes. 
+
+`nodeSelector` is a field of PodSpec. It specifies a map of key-value pairs. For the pod to be eligible to scheduled on a node, the node must have each of the indicated key-value pairs as labels (it can have additional labels as well). 
+
+
+When working with with `nodeSelector`, the indicated key-value pair will be matched to a node label.  
```

Comment:
```suggestion
When working with `nodeSelector`, the indicated key-value pair will be matched to a node label.  
```

Reply:
Thanks, this is auto-generated from [original policy](https://github.com/weaveworks/policy-library/blob/main/examples/ControllerAffinityNodeSelector/policy.yaml#L9).